packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -302,3 +317,15 @@ function proxyESM(mod: any) {
 function isPrimitive(value: any) {
   return !value || (typeof value !== 'object' && typeof value !== 'function')
 }
+
+function getErrorDepsStack(
+  urlStack: string[],
+  errorStack: WeakMap<string[], { importee: string }> | undefined,
+) {
+  if (errorStack) {
+    const { importee } = errorStack.get(urlStack)!
+    urlStack = urlStack.concat(importee)
+  }
+
+  return urlStack.filter((dep) => !dep.startsWith('virtual:'))
```

Comment:
Not sure if we need to filter by `virtual:`, it feels like important information to keep. Maybe we should not filter for now and see if it works later.

Reply:
`virtual:` should definitely be part of the stack: many frameworks including vite-plugin-ssr and Rakkas use virtual modules as entry. Keeping them is paramount.

Reply:
Make sense to me. Done.  Description screenshot updated